external/CapnProto.cmake

```diff
@@ -30,12 +30,14 @@ if (NOT ${SOURCE_CAPNP})
   find_library(LIB_CAPNPC ${STATIC_PRE}capnpc${STATIC_SUF})
   set(CAPNP_LIBRARIES ${LIB_CAPNPC} ${LIB_CAPNP} ${LIB_KJ})
   set(CAPNP_LIBRARIES_LITE ${LIB_CAPNP} ${LIB_KJ})
+  set(CAPNP_EXECUTABLE ${BIN_PRE}/capnp${CMAKE_EXECUTABLE_SUFFIX})
```

Comment:
They don't seem to be found with find_package when another ExternalProject installs pre-built tools.